zulipterminal/model.py

```diff
@@ -741,7 +743,8 @@ def _handle_message_event(self, event: Event) -> None:
         if 'read' not in message['flags']:
             set_count([message['id']], self.controller, 1)
 
-        if hasattr(self.controller, 'view') and self.found_newest:
+        if(hasattr(self.controller, 'view')
```

Comment:
I forgot the spacing.
Code block aligning?

Reply:
See how `and` and `if` get aligned if you add the space.
```python3
        if (hasattr(self.controller, 'view')
            and self.found_newest[repr(self.narrow)]):
            if self.msg_list.log:
```
This would trigger a linting error which you could fix with either:
```python3
        if (hasattr(self.controller, 'view')
                and self.found_newest[repr(self.narrow)]):
            if self.msg_list.log:
```
or
```python3
        if (
            hasattr(self.controller, 'view')
            and self.found_newest[repr(self.narrow)]
           ):
            if self.msg_list.log:
```

Reply:
Ah, I didn't know that thanks :+1: